config/jobs/kubernetes/sig-k8s-infra/trusted/sig-testing-label-sync.yaml

```diff
@@ -0,0 +1,42 @@
+periodics:
+- cron: "17 * * * *"  # Every hour at 17 minutes past the hour
+  name: ci-test-infra-label-sync
+  cluster: k8s-infra-prow-build-trusted
+  labels:
+    app: label-sync
+  decorate: true
+  spec:
+    containers:
+    - name: label-sync
+      image: gcr.io/k8s-prow/label_sync:v20240513-a9bd71bf01
+      command:
+      - label_sync
+      args:
+      - --config=/etc/config/labels.yaml
+      - --confirm=true
+      - --orgs=kubernetes,kubernetes-client,kubernetes-csi,kubernetes-sigs
+      - --token=/etc/github/token
+      - --endpoint=http://ghproxy.default.svc.cluster.local
+      - --endpoint=https://api.github.com
+      - --debug
+      volumeMounts:
+      - name: github
+        mountPath: /etc/github
+        readOnly: true
+      - name: config
+        mountPath: /etc/config
+        readOnly: true
+    volumes:
+    - name: github
+      secret:
+        secretName: k8s-github-robot-github-token
+    - name: config
+      configMap:
+        name: label-config
+  annotations:
+    testgrid-num-failures-to-alert: '6'
+    testgrid-alert-stale-results-hours: '12'
+    testgrid-dashboards: sig-testing-misc
+    testgrid-tab-name: label_sync
+    testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com, k8s-infra-oncall@google.com
```

Comment:
contribex leads instead ?

Reply:
yes, +1 to copying alerts to `kubernetes-sig-contribex-leads@` as well, in addition.

Removing `sig-testing` and `k8s-infra` DLs, I'm not sure on that part.
Is there's any historical context for these? cc: @cblecker @BenTheElder

Reply:
> Removing sig-testing and k8s-infra DLs, I'm not sure on that part.
> Is there's any historical context for these? cc: @cblecker @BenTheElder

SIG Testing has historically implemented and owned the code (... and this whole repo), versus the definition of the labels.

I'm going to leave this to a future PR, we do not need to block migrating off of the old cluster on this.

Reply:
If the job fails, it's been either the code => sig-testing, or an issue with prow / the trusted cluster => k8s infra or sig testing.

We also don't allow other SIGs to run arbitrary code in the trusted cluster generally, expanding permissions over that cluster is a security nightmare, but I wouldn't be opposed to moving this out to a cloudbuild job or cluster and moving the code out to another repo, similar to peribolos.

Reply:
Feel free to update the alert emails in a future PR, I don't want to keep making non-migration modifications because the migration work is already taking too long and pushing that change will delay merging and making sure the migration works.

Reply:
I filed a follow-up in https://github.com/kubernetes/test-infra/pull/32970/files